### PR TITLE
improved description

### PR DESCRIPTION
**Pull request by bvlucretiu**
That test change in the lesson with "Mersi"

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -96,16 +96,10 @@ <h3>Răzvan Cîrlugea</h3>
                 <div>
                   <h3>Full Name</h3>
                   <p>
-                    Lorem ipsum dolor sit amet consectetur, adipisicing elit.
-                    Molestiae omnis nobis quo dicta culpa eum exercitationem
-                    provident ex iure pariatur? Tempora sunt accusamus tempore
-                    modi.
+                    Lorem ipsum dolor sit amet consectetur, adipisicing elit. Molestiae omnis nobis quo dicta culpa eum exercitationem provident ex iure pariatur? Tempora sunt accusamus tempore modi.
```
Comment (razvancir96) [inline index.html]: Please remove this change, as it is not necessary

File: index.html [hunk 2]
```diff
@@ -96,16 +96,10 @@ <h3>Răzvan Cîrlugea</h3>
...
                   </p>
                   <p>
-                    Quidem, corrupti perspiciatis aliquid dicta cum odio
-                    sapiente tempora quas incidunt adipisci alias praesentium et
-                    excepturi error maxime aspernatur sint qui doloremque
-                    distinctio. Ex, unde.
+                    Quidem, corrupti perspiciatis aliquid dicta cum odio sapiente tempora quas incidunt adipisci alias praesentium et excepturi error maxime aspernatur sint qui doloremque distinctio. Ex, unde.
```
Comment (razvancir96) [inline index.html]: Please remove this change, as it is not necessary

File: index.html [hunk 3]
```diff
@@ -124,30 +118,27 @@ <h3>Full Name</h3>
...
-                    distinctio. Ex, unde.
+                    <i>"If you want to live a happy life,
+                    connect with a purpose, not people or objects!"</i>
+                    so my goal is JavaScript!<br>
+                    <i>"Problems to be had because there are solutions!"</i>
                   </p>
                 </div>
                 <div class="contact">
-                  <a href="#"
+                  <a target="blank" href="https://www.linkedin.com/in/vasilebirta/"
```
Comment (razvancir96) [inline index.html]: Please remove the target="blank" attribute, as I will have to add it for all links. Thank you for pointing this out! 🙏

File: index.html [hunk 4]
```diff
@@ -162,16 +153,10 @@ <h3>Full Name</h3>
                 <div>
                   <h3>Full Name</h3>
                   <p>
-                    Lorem ipsum dolor sit amet consectetur, adipisicing elit.
-                    Molestiae omnis nobis quo dicta culpa eum exercitationem
-                    provident ex iure pariatur? Tempora sunt accusamus tempore
-                    modi.
+                    Lorem ipsum dolor sit amet consectetur, adipisicing elit. Molestiae omnis nobis quo dicta culpa eum exercitationem provident ex iure pariatur? Tempora sunt accusamus tempore modi.
```
Comment (razvancir96) [inline index.html]: Please remove this change, as it is not necessary

File: index.html [hunk 5]
```diff
@@ -162,16 +153,10 @@ <h3>Full Name</h3>
...
                   </p>
                   <p>
-                    Quidem, corrupti perspiciatis aliquid dicta cum odio
-                    sapiente tempora quas incidunt adipisci alias praesentium et
-                    excepturi error maxime aspernatur sint qui doloremque
-                    distinctio. Ex, unde.
+                    Quidem, corrupti perspiciatis aliquid dicta cum odio sapiente tempora quas incidunt adipisci alias praesentium et excepturi error maxime aspernatur sint qui doloremque distinctio. Ex, unde.
```
Comment (razvancir96) [inline index.html]: Please remove this change, as it is not necessary

File: index.html [hunk 6]
```diff
@@ -124,30 +118,27 @@ <h3>Full Name</h3>
...
+                    Lifetime is a broken challenge in several episodes and IT & JavaScript is a passion that covers all other episodes especially since this passion has taken shape through the prism of vectors like <strong> <a style="text-decoration: none;" href="https://www.itschool.ro/"> ITSchool</a></strong> who have a team of high value instructors who do their job with great passion and one of them is <strong>Răzvan Cîrlugea</strong>, through whom I consider that I have acquired essential knowledge both in JavaScript and in teamwork through Git and GitHub!
                   </p>
                   <p>
-                    Quidem, corrupti perspiciatis aliquid dicta cum odio
-                    sapiente tempora quas incidunt adipisci alias praesentium et
-                    excepturi error maxime aspernatur sint qui doloremque
-                    distinctio. Ex, unde.
+                    <i>"If you want to live a happy life,
+                    connect with a purpose, not people or objects!"</i>
+                    so my goal is JavaScript!<br>
```
Comment (razvancir96) [inline index.html]: Do not use the `<br>` tag. If you want a new line, add a new `<p>` tag.

File: index.html [hunk 7]
```diff
@@ -124,30 +118,27 @@ <h3>Full Name</h3>
...
-                    provident ex iure pariatur? Tempora sunt accusamus tempore
-                    modi.
+                    Lifetime is a broken challenge in several episodes and IT & JavaScript is a passion that covers all other episodes especially since this passion has taken shape through the prism of vectors like <strong> <a style="text-decoration: none;" href="https://www.itschool.ro/"> ITSchool</a></strong> who have a team of high value instructors who do their job with great passion and one of them is <strong>Răzvan Cîrlugea</strong>, through whom I consider that I have acquired essential knowledge both in JavaScript and in teamwork through Git and GitHub!
                   </p>
                   <p>
-                    Quidem, corrupti perspiciatis aliquid dicta cum odio
-                    sapiente tempora quas incidunt adipisci alias praesentium et
-                    excepturi error maxime aspernatur sint qui doloremque
-                    distinctio. Ex, unde.
+                    <i>"If you want to live a happy life,
```
Comment (razvancir96) [inline index.html]: It's better to use the `<em>` tag instead of `<i>`

File: index.html [hunk 8]
```diff
@@ -124,30 +118,27 @@ <h3>Full Name</h3>
...
                 </div>
                 <div>
-                  <h3>Full Name</h3>
+                  <h3>Vasile Lucrețiu Birta</h3>
                   <p>
-                    Lorem ipsum dolor sit amet consectetur, adipisicing elit.
-                    Molestiae omnis nobis quo dicta culpa eum exercitationem
-                    provident ex iure pariatur? Tempora sunt accusamus tempore
-                    modi.
+                    Lifetime is a broken challenge in several episodes and IT & JavaScript is a passion that covers all other episodes especially since this passion has taken shape through the prism of vectors like <strong> <a style="text-decoration: none;" href="https://www.itschool.ro/"> ITSchool</a></strong> who have a team of high value instructors who do their job with great passion and one of them is <strong>Răzvan Cîrlugea</strong>, through whom I consider that I have acquired essential knowledge both in JavaScript and in teamwork through Git and GitHub!
```
Comment (razvancir96) [inline index.html]: `a style="text-decoration: none;" ` please remove this style, as all links should have the same style. Also, if we add style, we add it to a CSS file, not inline 😄

Comment (razvancir96) [inline index.html]: `with great passion and one of them is <strong>Răzvan Cîrlugea</strong>` The phrase becomes too long. You can split it here: `with great passion. One of them is <strong>Răzvan Cîrlugea</strong>`

**Review by razvancir96 — APPROVED** [review]

**Review by razvancir96 — CHANGES_REQUESTED** [review]
Sorry, I've approved the PR by mistake! Please read the comments for the requested changes

**Review by razvancir96 — APPROVED** [review]
Looking good now! 🚀

**Comment by bvlucretiu** [comment]
Thanks for appreciation!

**Pull request merged**